tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -1307,6 +1347,89 @@ def _reset_allow_build_at_runtime(node):
     signatures[self._input_saved_model_signature_key] = self._converted_func
     save.save(self._saved_model, output_saved_model_dir, signatures)
 
+  def summary(self, line_length=160, detailed=True, print_fn=None):
```

Comment:
This is a public API, you need to update RELEASE.md and the test golden file, similar to what you did to another PR.

Reply:
Fixed